etcdctl/ctlv3/command/check.go

```diff
@@ -67,6 +70,35 @@ var checkPerfCfgMap = map[string]checkPerfCfg{
 	},
 }
 
+type checkDatascaleCfg struct {
+	limit   int
+	kvSize  int
+	clients int
+}
+
+var checkDatascaleCfgMap = map[string]checkDatascaleCfg{
+	"s": {
+		limit:   10000,
+		kvSize:  1024,
+		clients: 50,
+	},
+	"m": {
+		limit:   100000,
+		kvSize:  1024,
+		clients: 200,
+	},
+	"l": {
+		limit:   1000000,
+		kvSize:  1024,
+		clients: 500,
+	},
+	"xl": {
+		limit:   30000000,
```

Comment:
why `30000000` but not `10000000`? I like the idea of increasing order of limit each time from `s -> m -> l -> xl`

Reply:
Thanks @fanminshi hat's like xxl :-) :-) I think we decided 3000000 considering this is a test and as we go higher in number it can take more time and resource. @xiang90 any thoughts?

Reply:
@spzala  I saw that `xl` = `3 * 10^7` and `l` = `1 * 10^6`.  So i am curious why isn't `xl` = `1*10^7` which is one order increase from `l`.

Reply:
xl tries to hit the upper bound aggressively which is 3 versions of 1M objects (which is 3M in total).

Reply:
@fanminshi ahhh..sorry I read one more zero in your suggestion and so was kidding with xxl :) Thanks @xiang90 for the explanation.

Reply:
@xiang90 make sense. maybe add a comment on xl to explain `3 versions of 1M objects `?

Reply:
@fanminshi sure, sounds good. Thanks!